verilog/formatting/token_annotator.cc

```diff
@@ -496,6 +496,17 @@ static WithReason<SpacingOptions> BreakDecisionBetween(
     return {SpacingOptions::MustWrap, "end* keywords should start own lines"};
   }
 
+  if (right.TokenEnum() == TK_else) {
+    if (left.TokenEnum() != TK_end)
+      return {SpacingOptions::MustWrap, "'else' token should start its own line unless preceded by 'end' without label."};
+    else
+      return {SpacingOptions::MustAppend, "'end'-'else' tokens should be together on one line."};
+  }
+
+  if ( (left.TokenEnum() == TK_else) && (right.TokenEnum() == TK_begin) ) {
+    return {SpacingOptions::MustAppend, "'else'-'begin' tokens should be together on one line."};
+  }
+
```

Comment:
Just for your information, these decisions here are bound in that it doesn't let the wrap optimizer decide how to wrap.  Perhaps this is a good thing in that it constrains the search.
No change needed.

Reply:
Okay. Thanks for info.